extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/YoutubeParsingHelper.java

```diff
@@ -73,8 +66,13 @@
 import java.util.Random;
 import java.util.regex.Pattern;
 
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import static org.schabi.newpipe.extractor.NewPipe.getDownloader;
```

Comment:
Why does your editor keep moving these around?

Reply:
Looks like IntelliJ uses a different code style than Android studio (despite being based on the same IDE):
![grafik](https://user-images.githubusercontent.com/40789489/183288443-0e05db0a-58bb-4815-8554-f824cf5f7a71.png)


I will reconfigure my IntelliJ so that it uses Android Studio's settings.

Maybe we should also track these settings in Git so that they are consistent between developers.

Reply:
For now I used Android Studios code format, since it's already used in the files.

Reply:
> Maybe we should also track these settings in Git so that they are consistent between developers.

Yeah, good idea. Could you do that, since you already looked into how it works? Thanks

Reply:
Not in this PR but I created an issue: https://github.com/TeamNewPipe/NewPipeExtractor/issues/892